tslint.json

```diff
@@ -1,18 +0,0 @@
-{
```

Comment:
This file should not be deleted nor renamed to "tslint 2.json`.  Any reason to do so?

Reply:
i think i haven't deleted nor renamed that file, but i've ```rm -rf node_modules``` and ```npm i``` again. Should i renamed the file like before ?

Reply:
Yes.